nncf/helpers/dataset.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2024 Intel Corporation
```

Comment:
I would suggest to move this file to `nncf/data/generators.py`

Reply:
Why `generators`? How it would correlate with the requirement to have `from nncf.data import generate_text_data` import?

Reply:
It does not influence to the requirement to have from `nncf.data import generate_text_data` import. Could you make a proposal taking account removing `helpers` folder?

Reply:
No proposals anymore.